src/aind_dynamic_foraging_basic_analysis/metrics/trial_metrics.py

```diff
@@ -1,19 +1,23 @@
 """
     Tools for computing trial by trial metrics
-    df_trials = compute_all_trial_metrics(nwb)
+    df_trials = compute_trial_metrics(nwb)
```

Comment:
output of this function is `df` not `df_trials`

Reply:
I updated the code to clarify that all the dataframes are `df_trials`, I just used `df` in places as a shorthand.